jerry-core/jerry.c

```diff
@@ -635,6 +635,24 @@ jerry_api_release_object (jerry_api_object_t *object_p) /**< pointer acquired th
 } /* jerry_api_release_object */
 
 /**
+ * Acquire specified Jerry API value
+ */
+jerry_api_value_t *
+jerry_api_acquire_value (jerry_api_value_t *value_p) /**< API value */
+{
+  jerry_assert_api_available ();
+
+  if (value_p->type == JERRY_API_DATA_TYPE_STRING)
+  {
+    jerry_api_acquire_string (value_p->u.v_string);
+  }
+  else if (value_p->type == JERRY_API_DATA_TYPE_OBJECT)
+  {
+    jerry_api_acquire_object (value_p->u.v_object);
+  }
```

Comment:
I humbly disagree. Internal-only functions can be vague about implementation details (even though it's not ideal but that's how it works in practice) but this is the _public_ API of jerry. It _may_ be a no-op for everything beyond strings and objects but it _must_ document this. (My comment above also mentioned "nor does it document".)

And true enough, `jerry_api_release_value` has a deficiency in its documentation. This is no justification for making the same mistake in a new function. Better to get the new function right _and_ fix the existing doc as well.